src/cirrocumulus/scripts/cirro_scanner.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# cirro_scanner.sh scans the mounted buckets to find cirro data. Creates a cirro dataset when *.zarr folder
+# is found and is not yet in the existing cirro datasets.
+
+if [[ $# -ne 1 ]]; then
+  echo "Usage: $0 <true/false>"
```

Comment:
```suggestion
  echo "Usage: $0 <true|false>"
```

Reply:
Please add a usage description to the top level script comment such that the caller does not need to read the code below to figure out what true|false represent.